go.mod

```diff
@@ -1,45 +1,21 @@
 module github.com/bbva/qed
 
+go 1.12
```

Comment:
go 1.12 is required?

Reply:
Go 1.11.4 seems to work too with that line. I suppose the modules thing is going to change these days in preparation to the final release in go 1.13 this summer.